pkg/test/environment.go

```diff
@@ -67,9 +67,17 @@ type EnvironmentOption func(env *Environment)
 
 func NewEnvironment(ctx context.Context, options ...EnvironmentOption) *Environment {
 	ctx, stop := context.WithCancel(ctx)
+	customApiServerFlags := []string{
```

Comment:
This creates a dual-stack cluster.  Which is fine, but I should point out that it is not the same as an EKS IPv6 cluster.  (EKS is an _IPv6-only_ cluster, with dual-stack pods)